src/søknad/steg/4-barnasbosted/AnnenForelderKnapper.tsx

```diff
@@ -50,47 +51,45 @@ const AnnenForelderKnapper: React.FC<Props> = ({
   };
 
   const leggTilAnnenForelder = () => {
-    settHuketAvAnnenForelder(false);
+    settBarnHarSammeForelder(false);
```

Comment:
Endrer navn på metode og variabel siden det ikke er en checkbox det gjelder, men en sjekk for å sjekke om barn 2+ har samme forelder som barn 1 (slik at de skal slippe å fylle ut alle spm&felter på nytt) eller barn 2+ har en annen forelder.


![Skjermbilde 2020-06-02 kl  09 45 32](https://user-images.githubusercontent.com/8249453/83493933-d357bd80-a4b5-11ea-83a5-bd5db8c0cfcb.png)